Makefile

```diff
@@ -51,76 +55,79 @@ main-cuda: main_cuda.cc rawtodigi_cuda.cu rawtodigi_cuda.h
 debug-cuda: main_cuda.cc rawtodigi_cuda.cu rawtodigi_cuda.h
 	$(NVCC) $(NVCC_FLAGS) -DDIGI_CUDA $(NVCC_DEBUG) -o $@ main_cuda.cc rawtodigi_cuda.cu
 
+ifdef ALPAKA_BASE
 # Alpaka/cupla implementation, with the CUDA GPU async backend
-cupla-cuda-async: main-cupla-cuda-async
+cupla: main-cupla-cuda-async
```

Comment:
To me
```makefile
cupla: main-cupla-cuda-async main-cupla-seq-seq-async main-cupla-seq-seq-sync main-cupla-tbb-seq-async main-cupla-omp2-seq-async
```
would be more clear than the scattered definitions of `cupla` target.

Reply:
The idea was to make it simple to add more "cupla" targets, without having to update a single central place.
Of course for "all" we do just that, so we can be more consistent and have a single entry for "cupla" as well.

Reply:
Right, on one hand it would be more useful to avoid a central place (conflicts between developers etc). In the end I don't feel strongly about this, I can grep the Makefile now that I know what to look for.